src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -137,12 +137,7 @@ public void FetchJitTools(string runtime, string restoreFolder)
                 throw;
             }
 
-            if (jitExecutablePath == null)
-            {
-                throw new InvalidOperationException($"Cannot find executable {jitExecutable}.");
-            }
-
-            _clrJitPath = jitExecutablePath;
+            _clrJitPath = jitExecutablePath ?? throw new InvalidOperationException($"Cannot find executable {jitExecutable}.");
```

Comment:
The earlier code did the same. It is just short hand for what earlier code used to do.